README.md

```diff
@@ -1,3 +1,95 @@
-# Ansible Collection - cloud.gcp_ops
+# cloud.gcp_ops roles/playbooks to demo Ansible on AWS
 
-Documentation for the collection.
+This repository hosts the `cloud.gcp_ops` Ansible Collection.
+
+The collection includes a variety of Ansible roles and playbooks to help automate the management of resources on GCP.
+
+<!--start requires_ansible-->
+## Ansible version compatibility
+
+This collection has been tested against following Ansible versions: **>=2.12.0**.
+
+## Included content
+
+Click on the name of a role to view that content's documentation:
+
+<!--start collection content-->
+### Roles
+Name | Description
+--- | ---
+[cloud.gcp_ops.manage_machine_image](https://github.com/ansible-collections/cloud.aws_ops/blob/main/roles/manage_machine_image/README.md)|A role to create / delete a GCP custom machine image.
```

Comment:
This should be added in another PR once the role is merged

Reply:
Removed!